### DISCUSSION TITLE
How to Change TTS API Response Format from String to Audio File (.mp3)

### DISCUSSION THREAD

**Discussion by teneous** [langgenius/dify#8715] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Hi，I have a TTS API `http://127.0.0.1:17861/v1/tts/json` which makes user's text to audio. And the request format is `application/json`, the reponse is `audio/mpeg`.

This is my workflow image
<img width="1148" alt="image" src="https://github.com/user-attachments/assets/cd1b7c71-052c-4e5e-b551-f87cdb139320">

This is HTTP content
<img width="401" alt="image" src="https://github.com/user-attachments/assets/e282731b-d064-4fe2-bc83-f432b2135ee1">

As you can see, the body return string, it's so wired, how can I change it to audio(.mp3).

**Comment by backearth1** [comment]
same requirement
